dspace-xmlui/src/main/webapp/themes/Mirage/lib/xsl/aspect/artifactbrowser/item-view.xsl

```diff
@@ -40,10 +40,10 @@
 
     <xsl:template name="itemSummaryView-DIM">
         <!-- optional: Altmeric.com badge and PlumX widget -->
-        <xsl:if test='confman:getProperty("altmetrics", "altmetric.enabled") and ($identifier_doi or $identifier_handle)'>
+        <xsl:if test="confman:getProperty('altmetrics', 'altmetric.enabled') = 'true' and ($identifier_doi or $identifier_handle)">
```

Comment:
This PR will need to be updated for the enhanced Configuration system (see #1104).

The `confman:getProperty` calls all call the ConfigurationManager. Now, if you pass it two params, they are just appended together. So, this call will end up trying to load a property named `altmetrics.altmetric.enabled` from `altmetrics.cfg`. The correct property is simply `altmetric.enabled`